robots/gazebo_tank.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+// BoB robotics includes
+#include "../common/pose.h"
+#include "../common/stopwatch.h"
+#include "tank.h"
+
+// Third-party includes
+#include "../third_party/units.h"
+#include <utility> 
+namespace BoBRobotics {
+namespace Robots {
+using namespace units::literals;
+
+template<typename LengthUnit = units::length::millimeter_t,
+         typename AngleUnit = units::angle::degree_t>
+class GazeboTank
+  : public Tank
+{
+    using millimeter_t = units::length::millimeter_t;
+    using meters_per_second_t = units::velocity::meters_per_second_t;
+
+public:
+    GazeboTank(const meters_per_second_t maximumSpeed)
+      : m_MaximumSpeed(maximumSpeed)
+    {}
+
+
+    const auto &getWheelSpeeds(){
+        return m_WheelSpeeds;
+    }
```

Comment:
This function should be marked ``const``.

Reply:
Actually, this function is no longer required since I've moved the driving logic into GazeboTank class. This function was earlier being used by simulated_tank_gazebo to set the wheel speed messages. Should we still keep it?

Reply:
Nah, let's drop it. We also don't need to store ``m_WheelSpeeds`` any more.